packages/ContentAutotagging/LocalFileSystem/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@memberjunction/content-autotagging-localfilesystem",
+    "version": "1.0.0",
+    "description": "MemberJunction Content Autotagging Local File System Package",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "/dist"
+    ],
+    "scripts": {
+        "build": "tsc -p tsconfig.json",
+        "start": "ts-node-dev src/index.ts",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "MemberJunction.com",
+    "license": "ISC",
+    "dependencies": {
+        "@memberjunction/ai": "2.2.1",
+        "@memberjunction/global": "2.2.1",
+        "@memberjunction/content-autotagging-core": "1.0.0",
+        "@memberjunction/content-autotagging-engine": "1.0.0",
+        "date-fns": "^3.6.0",
+        "moment": "^2.30.1",
+        "moment-timezone": "^0.5.45"
```

Comment:
Why use both moment and date-fns? Prob best to pick a lib and stick with it. Also Moment is not generally recommended for new projects after 2020:

> Considering using Moment in your project?
There may be better modern alternatives.
For more details and recommendations,
please see [Project Status](https://momentjs.com/docs/#/-project-status/) in the docs.